optuna/study.py

```diff
@@ -778,19 +776,18 @@ def _run_trial(
             TypeError,
         ):
             message = (
-                "Setting status of trial#{} as {} because the returned value from the "
-                "objective function cannot be casted to float. Returned value is: "
-                "{}".format(trial_number, TrialState.FAIL, repr(result))
+                "Trial {} failed, because the returned value from the "
+                "objective function cannot be cast to float. Returned value is: "
+                "{}".format(trial_number, repr(result))
             )
             _logger.warning(message)
             self._storage.set_trial_system_attr(trial_id, "fail_reason", message)
             self._storage.set_trial_state(trial_id, TrialState.FAIL)
             return trial
 
         if math.isnan(result):
-            message = (
-                "Setting status of trial#{} as {} because the objective function "
-                "returned {}.".format(trial_number, TrialState.FAIL, result)
+            message = "Trial {} failed, because the objective function " "returned {}.".format(
```

Comment:
Implicit string concat.
```suggestion
            message = "Trial {} failed, because the objective function returned {}.".format(
```